posthog/ai/langchain/callbacks.py

```diff
@@ -452,10 +467,14 @@ def _get_langchain_run_name(self, serialized: Optional[Dict[str, Any]], **kwargs
 
     def _capture_trace(self, run_id: UUID, *, outputs: Optional[Dict[str, Any]]):
         trace_id = self._get_trace_id(run_id)
+        run = self._pop_run_metadata(run_id)
```

Comment:
Pretty sneaky to pop in a `_capture_...` method, how about renaming it to `_pop_trace_and_capture` for clarity? Alternatively, we can just do `self._runs[run_id]` here and pop outside of here